dspace-api/src/main/java/org/dspace/health/Check.java

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Please use the verbatim license header to comply with our automated license header. Move your credits down to a separate comment. Use "mvn package license:check -Dmaven.test.skip=false" to test.

[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/Check.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/Report.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/UserCheck.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/LogAnalyserCheck.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/ItemCheck.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/InfoCheck.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/ChecksumCheck.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/EmbargoCheck.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/Core.java
[WARNING] Missing header in: /home/travis/build/DSpace/DSpace/dspace-api/src/main/java/org/dspace/health/ReportInfo.java